### DISCUSSION TITLE
Where is Theme-UI's "default" theme?

### DISCUSSION THREAD

**Discussion by sheedy** [system-ui/theme-ui#1157] [General] [answered]
A default theme seems to exist but I can not for the life of me find out where it is being imported/pulled from.

If I set my theme as an empty object and provide it to `ThemeProvider`, I'm still able to use spacing scales in props like `mb={[ 2, 4 ]}`.

I've looked and searched the entire code base (and parts of Emotion's) and can not figure out what is defining the default theme values.

Can anyone point me to it and or explain how to "reset" it?

**Comment by hasparus** [comment, marked as answer]
It's in `packages/css/src/index`

https://github.com/system-ui/theme-ui/blob/16a048134b2dd13b6c23a00748d44606ef25081b/packages/css/src/index.js#L8-L15

**Reply by sheedy** [reply]
Thanks @hasparus! I remember checking that file and skimming over it because I was expecting to see more than those 2 values.
